tests/basic_api_test.sh

```diff
@@ -89,10 +93,69 @@ curl -L -X POST "http://$QDRANT_HOST/collections" \
       }
   }' | jq
 
+# search points
 curl -L -X POST "http://$QDRANT_HOST/collections/test_alias/points/search" \
   -H 'Content-Type: application/json' \
   --fail -s \
   --data-raw '{
         "vector": [0.2,0.1,0.9,0.7],
         "top": 3
     }' | jq
+
+# delete points
+curl -L -X POST "http://$QDRANT_HOST/collections/test_collection/points/delete?wait=true" \
+  -H 'Content-Type: application/json' \
+  --fail -s \
+  --data-raw '[ 1, 2, 3, 4, 5 ]' | jq
```

Comment:
It might be better to make schema for this request with explicit name for ids like

```json
{
  "ids": [ 1, 2, 3, 4, 5 ]
}
```

What do you think?

It could be a query later..

Reply:
I agree :+1: